src/components/Kv/KvMap.vue

```diff
@@ -308,6 +398,82 @@ export default {
 				this.createWrapperObserver();
 			}
 		},
+		countryStyle(feature) {
+			return {
+				color: kvTokensPrimitives.colors.white,
+				fillColor: this.getCountryColor(feature.lenderLoans),
+				weight: 1,
+				fillOpacity: 1,
+			};
+		},
+		getCountryColor(lenderLoans) {
+			const loanCountsArray = [];
+			this.countriesData.forEach(country => {
+				loanCountsArray.push(country.value);
+			});
+
+			const maxNumLoansToOneCountry = Math.max(...loanCountsArray);
+			const intervals = getIntervals(1, maxNumLoansToOneCountry, 6);
+
+			if (intervals.length === 1) {
+				const [inf, sup] = intervals[0]; // eslint-disable-line no-unused-vars
+
+				for (let i = 0; i < sup; i += 1) {
+					const loansNumber = i + 1;
+
+					if (lenderLoans && lenderLoans >= loansNumber && lenderLoans < loansNumber + 1) {
+						return kvTokensPrimitives.colors.brand[mapColors[i]];
+					}
+				}
+			} else {
+				for (let i = 0; i < intervals.length; i += 1) {
+					const [inf, sup] = intervals[i];
+					if (lenderLoans && lenderLoans >= inf && lenderLoans <= sup) {
+						return kvTokensPrimitives.colors.brand[mapColors[i]];
+					}
+				}
+			}
+
+			return kvTokensPrimitives.colors.gray[300];
+		},
+		onEachCountryFeature(feature, layer) {
+			const loansString = feature.lenderLoans
+				? `${feature.lenderLoans} loan${feature.lenderLoans > 1 ? 's' : ''}`
+				: '0 loans';
+			const countryString = `${feature.properties.ADMIN} <br/> ${loansString}`;
+
+			layer.bindTooltip(countryString, {
+				sticky: true,
+			});
+
+			layer.on({
+				mouseover: this.highlightFeature,
+				mouseout: this.resetHighlight,
+			});
+		},
+		highlightFeature(e) {
+			const layer = e.target;
+
+			layer.setStyle({
+				fillColor: kvTokensPrimitives.colors.gray[500],
+			});
+		},
+		resetHighlight(e) {
+			const layer = e.target;
+			const { feature } = layer;
+
+			layer.setStyle({
+				fillColor: this.getCountryColor(feature.lenderLoans),
+			});
+		},
+		circleMapClicked(countryIso) {
+			this.$router.push({
```

Comment:
Should we also be emitting here so we can send analytics events? Or call analytics here directly?

Reply:
Yeah, for this scope we have other ticket for analytics events... but if we port this feature to the shared library, we need to add an emit for this.